lib/scholar/linear/bayesian_ridge_regression.ex

```diff
@@ -428,6 +428,8 @@ defmodule Scholar.Linear.BayesianRidgeRegression do
 
   @doc """
   Makes predictions with the given `model` on input `x`.
+  Output predictions have shape {n_samples} when train target is shaped either {n_samples} or {n_samples, 1}.
```

Comment:
We should start a new paragraph, because the first paragraph should always be a short summary. And use backticks around code, can you please apply similar changes below? :)


```suggestion

  Output predictions have shape `{n_samples}` when train target
  is shaped either `{n_samples}` or `{n_samples, 1}`.
```

Reply:
Sure thing. Thank you for your comments.